mobile/src/main/res/values/strings.xml

```diff
@@ -72,13 +72,13 @@
     <string name="error_network_not_available">Network is not available</string>
     <string name="error_http_connection_failed">Received an invalid response from openHAB. Please check if openHAB is working correctly (HTTP response code %d)</string>
     <string name="error_no_uuid">Unable to get openHAB UUID, connection failed</string>
-    <string name="error_invalid_url">Please enter a valid URL in a \'protocol://host:port/\' form</string>
+    <string name="error_invalid_url">Please enter a valid URL in a \‘http(s)://host(:port)/\’ form</string>
```

Comment:
Attention: You've inserted non-neutral single-quotes

Reply:
The quote change was suggested by Android Studio. Not sure if it's really better

Reply:
Okay... Could you find out the reasoning for that? What are these quotes normally used for? Do they need to be escaped? I'd vote to go back to the old quotes or to unicode-escape them.

Reply:
I reverted the change. Does double quotes `"` suit better here?

Reply:
Single quotes seem normal for these kinds of technical strings. Would be okay with me.